mtp_noms_ops/assets-src/javascripts/modules/security-forms.js

```diff
@@ -56,6 +56,9 @@ exports.SecurityForms = {
 
         $creditList.find('caption').remove();
         $creditList.find('.CollapsingTable').removeClass('CollapsingTable');
+        $creditList.find('tr th:first-of-type, tr td:first-of-type').each(function() {
+          $(this).remove();
+        });
         $thisRow.find('td').each(function () {
```

Comment:
i figured jquery would deal with that, it doesn't use native selector mechanisms does it?